<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>author_detectorClass</class>
 <widget class="QMainWindow" name="author_detectorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>author_detector</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>531</width>
      <height>381</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Train/Classify</string>
     </attribute>
     <widget class="QPushButton" name="browseDirButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Directory for train/classify</string>
      </property>
     </widget>
     <widget class="QPushButton" name="classifyButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>220</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Classify Directory</string>
      </property>
     </widget>
     <widget class="QPushButton" name="trainButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Train</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dirLineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>401</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>trainingSet/</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Model file for save/load</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="modelLineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>401</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>model.txt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browseModelFileButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>File with true text's authors</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="trueLineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>401</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>testingSet/answer.txt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browseTrueFile">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>130</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>File with predicted text's authors</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="predictedLineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>401</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>predicted.txt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browsePredictedFile">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>180</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="estimateButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>220</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Estimate Quality</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Precision:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>290</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>330</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Zhelubenkov Alexandr, 2013</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>143</x>
        <y>210</y>
        <width>48</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Количество триграмм</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Text Detector</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Model file</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browseModelFileButton_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="modelLineEdit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>401</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>model.txt</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>File with text for prediction</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browseTextFile">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textLineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>401</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>401</width>
        <height>171</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(251, 255, 201);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="predictButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>130</y>
        <width>75</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Predict</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Author:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>300</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>330</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Zhelubenkov Alexandr, 2013</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="author_detector.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>browseDirButton</sender>
   <signal>clicked()</signal>
   <receiver>author_detectorClass</receiver>
   <slot>onDirBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseModelFileButton</sender>
   <signal>clicked()</signal>
   <receiver>author_detectorClass</receiver>
   <slot>onModelFileBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseTrueFile</sender>
   <signal>clicked()</signal>
   <receiver>author_detectorClass</receiver>
   <slot>onTrueFileBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browsePredictedFile</sender>
   <signal>clicked()</signal>
   <receiver>author_detectorClass</receiver>
   <slot>onPredictedFileBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trainButton</sender>
   <signal>clicked()</signal>
   <receiver>author_detectorClass</receiver>
   <slot>onTrain()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>classifyButton</sender>
   <signal>clicked()</signal>
   <receiver>author_detectorClass</receiver>
   <slot>onClassify()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>estimateButton</sender>
   <signal>clicked()</signal>
   <receiver>author_detectorClass</receiver>
   <slot>onEstimate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseModelFileButton_2</sender>
   <signal>clicked()</signal>
   <receiver>author_detectorClass</receiver>
   <slot>onModelFileBrowse2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseTextFile</sender>
   <signal>clicked()</signal>
   <receiver>author_detectorClass</receiver>
   <slot>onTextFileBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>predictButton</sender>
   <signal>clicked()</signal>
   <receiver>author_detectorClass</receiver>
   <slot>onPredict()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSave()</slot>
  <slot>onModelFileBrowse2()</slot>
  <slot>onDirBrowse()</slot>
  <slot>onModelFileBrowse()</slot>
  <slot>onTrueFileBrowse()</slot>
  <slot>onPredictedFileBrowse()</slot>
  <slot>onTrain()</slot>
  <slot>onClassify()</slot>
  <slot>onEstimate()</slot>
  <slot>onTextFileBrowse()</slot>
  <slot>onPredict()</slot>
 </slots>
</ui>
